Dockerfile.alpine

```diff
@@ -15,7 +15,9 @@ WORKDIR e2fsprogs
 RUN git checkout fuse2fs-offset
 
 # Compile
-RUN mkdir libfuse && wget -Olibfuse/libfuse.a https://github.com/ruanformigoni/libfuse/releases/download/c9db3e5/libfuse.a
+# !! Add this back before merge !!
+# RUN mkdir libfuse && wget -Olibfuse/libfuse.a "https://github.com/ruanformigoni/libfuse/releases/download/c9db3e5/libfuse-$(uname -m).a"
+RUN mkdir libfuse && wget -Olibfuse/libfuse.a "https://github.com/hwittenborn/libfuse/releases/download/cf9319b/libfuse-$(uname -m).a"
```

Comment:
These need to be adjusted as needed after https://github.com/ruanformigoni/libfuse/pull/1 is merged.

Reply:
Ok, I'll do that.